packages/config/src/config.js

```diff
@@ -108,6 +110,50 @@ function resetConfig(oldConfig) {
   return clearConfig().then(config(oldConfig))
 }
 
+async function load(data) {
+  const network = await get("flow.network")
+  const cleanedNetwork = cleanNetwork(network)
+  const { flowJSON } = data
```

Comment:
Should we add an invariant to check that `flowJSON` is defined?

Reply:
Updated